extensions_admin/pulp_rpm/extensions/admin/copy_commands.py

```diff
@@ -93,7 +93,8 @@ def modify_user_input(self, user_input):
 
         # Work around to scope the fields loaded by the platform to limit the amount of
         # RAM used. This addition will find its way to the criteria parsing in the bindings.
-        user_input['fields'] = UNIT_KEY_RPM
+        # need the downloaded flag for later usage
+        user_input['fields'] = UNIT_KEY_RPM + ('downloaded',)
```

Comment:
we also need to extract the 'downloaded' field, because later it will be used during the copy operation